W-20727606-retrieve-asset-metadata-using-MCP-server-endpoint-KS (#349)
mulesoft/docs-exchange: modules/ROOT/pages/about-platform-apis.adoc

endif::[]
:imagesdir: ../assets/images/

> Anypoint Exchange provides API features for publishing and consuming assets by using the Exchange API, publishing and consuming assets by using the Exchange Maven Facade API, and searching for assets by using the Exchange Graph API. Use the Exchange MCP server endpoint to retrieve asset metadata for AI agents.

Reviewer: ```suggestion
Anypoint Exchange provides several APIs to help you manage assets:

* *Exchange API*: Publish and consume assets, and retrieve metadata for Agent Fabric assets.
* *Exchange Maven Facade API*: Publish and consume assets.
* *Exchange Graph API*: Search for assets.
```

Reviewer: A bulleted list makes it easier to scan.

* https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-experience-api/[Exchange API]
* https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/[Exchange Maven Facade API]
* https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-graph-http/[Exchange Graph API]

> * https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-experience-api/minor/2.4/pages/p1k-jaq/MCP%20Server/[Exchange MCP Server]

Reviewer: ```suggestion

```

Reviewer: Removed because it's not a standalone MCP server. We are documenting an MCP endpoint that acts as a bridge to allow AI models to interact with existing capabilities in the Exchange API.

Topics:

* xref:exchange-api.adoc[]
* xref:to-publish-assets-maven.adoc[]
* xref:to-search-with-graph-api.adoc[]

> * xref:retrieve-asset-metadata-using-MCP.adoc[]

Reviewer: ```suggestion
* xref:retrieve-asset-metadata-using-mcp.adoc[]
```

Reviewer: adoc file names must be lowercase. Make sure you rename the actual file name in all lowercase.
